pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -545,7 +545,7 @@ func parseAppInstanceConfig(config *zconfig.EdgeDevConfig,
 				adapter.Type, adapter.Name)
 			appInstance.IoAdapterList = append(appInstance.IoAdapterList,
 				types.IoAdapter{Type: types.IoType(adapter.Type),
-					Name: adapter.Name})
+					Name: adapter.Name, EthVf: types.EthVF{Mac: adapter.EthVf.Mac, Index: uint8(adapter.EthVf.VlanId)}})
```

Comment:
Beware that VF Index and the assigned VlanID is not the same thing:
`Index: uint8(adapter.EthVf.VlanId)`
Not even the data type matches (8 bits vs 16bits)